### DISCUSSION TITLE
What`s the difference between Shinchiro and yours fork?

### DISCUSSION THREAD

**Discussion by pingogeek** [zhongfly/mpv-build#28] [Q&A] [answered]
subj

**Comment by zhongfly** [comment, marked as answer]
almost identical, using the same configuration
You can simply think: Shinchiro set everything up, I use his settings without any modification.
Probably the only difference is the build time is different and I don't compile the 32 bit version

**Reply by zhongfly** [reply]
The original intention of this project was because the update interval of Shinchiro was long at that time, and I didn't want to configure the compilation environment on my computer, so I used github action, which is also convenient for others compile mpv at any time without any compilation knowledge or using your own computer. 
And now Shinchiro also uses github action to compile mpv every day.
